src/cpp/src/image_generation/stable_diffusion_3_pipeline.hpp

```diff
@@ -137,25 +137,17 @@ class StableDiffusion3Pipeline : public DiffusionPipeline {
 
         set_scheduler(Scheduler::from_config(root_dir / "scheduler/scheduler_config.json"));
 
-        // Temporary fix for GPU
```

Comment:
since this WA is not released yet (added to master after 2024.6), we can safely drop it and rely on official solution